src/pybind/mgr/rbd_support/mirror_snapshot_schedule.py

```diff
@@ -350,6 +370,12 @@ def run(self) -> None:
             self.log.info("MirrorSnapshotScheduleHandler: starting")
             while True:
                 refresh_delay = self.refresh_images()
+                if self.need_to_reconnect:
```

Comment:
Is `need_to_reconnect` flag actually needed?  Can `BlocklistedError` exception just be propagated and handled here?  A bit less state to manage that way.

Reply:
good point. Done.